mobile/src/assets/img/icons/home-icon/home-icon.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
this should be in svg format, to change color dynamically you can set fill=currentColor, so path will be assets/img/icons/home.svg